frontend/src/cards/TableCard.tsx

```diff
@@ -71,7 +71,7 @@ export function TableCard(props: TableCardProps) {
     <CardWrapper
       queries={[query]}
       title={
-        <>{`${props.variableConfig.variableFullDisplayName} By ${
+        <>{`${props.variableConfig.metrics.per100k.fullCardTitleName} By ${
```

Comment:
I don't think we should make this change, because the table shows more than just per 100k metrics.